scalableminds/webknossos
# Fix zarr private links for volume tracings and some minor improvements to private links UI

### URL of deployed dev instance (used for testing):
- https://fixzarrprivatelinksforvolumetracings.webknossos.xyz

### Steps to test:
- create some private links for an annotation and edit the expiration date to different dates to make sure the rendering makes sense

### Issues:
- fixes #6514
- contributes to #6474

------
(Please delete unneeded items, merge only when none are left open)
